gensim/models/base_any2vec.py

```diff
@@ -583,15 +940,58 @@ def _update_job_params(self, job_params, epoch_progress, cur_epoch):
         return next_alpha
 
     def _get_thread_working_mem(self):
+        """Computes the memory used per worker thread.
+
+        Returns
+        -------
+        (np.ndarray, np.ndarray)
+            Each worker threads private work memory.
```

Comment:
don't forget about empty line at the end of each docstring (except one-line docstrings), here and everywhere